<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmBarVolume</class>
 <widget class="QWidget" name="frmBarVolume">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="BarVolume" name="barVolume1"/>
   </item>
   <item>
    <widget class="BarVolume" name="barVolume2">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="BarVolume" name="barVolume3"/>
   </item>
   <item>
    <widget class="BarVolume" name="barVolume4"/>
   </item>
   <item>
    <widget class="BarVolume" name="barVolume5"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BarVolume</class>
   <extends>QWidget</extends>
   <header>barvolume.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
